checkstyle/import-control.xml

```diff
@@ -118,6 +118,7 @@
     <subpackage name="protocol">
       <allow pkg="org.apache.kafka.common.errors" />
       <allow pkg="org.apache.kafka.common.message" />
+      <allow pkg="org.apache.kafka.common.protocol" />
```

Comment:
Does this change take advantage of this? Couldn't find a usage of this.

Reply:
This change doesn't, but a future one needs the ability for the protocol package to import its own classes (I think for a static import)